namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/01-rbac.yaml

```diff
@@ -0,0 +1,16 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hmpps-contacts-dev-admin
+  namespace: hmpps-contacts-dev
+subjects:
+  - kind: Group
+    name: "github:dps-tech"
+    apiGroup: rbac.authorization.k8s.io
+  - kind: Group
+    name: "github:hmpps-move-and-improve"
+    apiGroup: rbac.authorization.k8s.io
```

Comment:
Are we all in GH group hmpps-move-and-improve?  Actually I'm not, but I can organise this.

Reply:
You and Rosario are.

Reply:
yeah you are not in this group yet, https://github.com/orgs/ministryofjustice/teams/hmpps-move-and-improve?query= please get your git account added to this group